daisy_integration_tests/scripts/post_translate_test.sh

```diff
@@ -219,10 +209,10 @@ function check_package_install {
 wait_for_connectivity
 
 # Run tests.
+check_package_install
```

Comment:
why reorder?

Reply:
shoot, I re-ordered during development to get the results faster. Reverted to the old order.